qt/aqt/addons.py

```diff
@@ -842,6 +842,7 @@ def onViewFiles(self) -> None:
 
     def onDelete(self) -> None:
         selected = self.selectedAddons()
+        gui_hooks.addons_dialog_will_delete_addon(self, selected)
```

Comment:
```suggestion
        gui_hooks.addons_dialog_will_delete_addons(self, selected)
```

Reply:
The hook should probably be run after the 'not selected' and 'askUser' lines, not before it.